velox/exec/tests/AggregationTest.cpp

```diff
@@ -1245,6 +1245,91 @@ TEST_F(AggregationTest, groupingSets) {
       "SELECT k1, k2, count(1), sum(a), max(b) FROM tmp GROUP BY ROLLUP (k1, k2)");
 }
 
+const std::shared_ptr<const core::GroupIdNode> findGroupIdNode(
```

Comment:
Hmm... If you need this functionality, consider adding `PlanBuilder::capturePlanNode(core::PlanNodePtr& planNode)` method.